lib/nerves/utils.ex

```diff
@@ -2,7 +2,18 @@ defmodule Nerves.Utils do
   @moduledoc false
   @alphanum 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789_'
 
-  @json_codec Jason
+  @json_codec Application.compile_env(:nerves, :json_codec) || Jason
+
+  unless Code.ensure_loaded?(@json_codec) do
+    Mix.raise("""
+    Nerves requires a JSON codec to be defined.
+
+    Please include :jason as a dependency or configure your own JSON parser
+    by updating your config.exs
+
+      config :nerves, json_codec: Poison
```

Comment:
```suggestion
      config :nerves, json_codec: Jason
```

Reply:
Jason is the default. This is just showing that it could be something different. The message could probably be updated entirely...